server/services/meross/utils.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * Generate MD5.
+ *
+ * @see http://www.webtoolkit.info/
+ * @param string - String to md5.
+ * @returns MD5 result.
+ * */
+function md5(string) {
```

Comment:
Maybe include https://www.npmjs.com/package/jshashes or https://www.npmjs.com/package/md5 ?

Reply:
You are right 👍 
It's done ;)